packages/twenty-front/src/modules/object-record/record-index/components/RecordIndexBoardDataLoader.tsx

```diff
@@ -45,6 +45,13 @@ export const RecordIndexBoardDataLoader = ({
           columnId={columnIds[index]}
         />
       ))}
+      <RecordIndexBoardColumnLoaderEffect
```

Comment:
I actually think this should be computed from the recordBoardcolumnDefinition array just to be consistent